From 6556e62a07972eafccaefdcdf32bd0d386c64ca2 Mon Sep 17 00:00:00 2001
From: Yegor Yefremov <yegorslists@googlemail.com>
Date: Wed, 13 Sep 2017 07:59:50 +0200
Subject: [PATCH] Declare mqtt-tests as a C-only project

This way CMake doesn't break the build on the systems without
C++ compiler.

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>
Reported-by: Marcus Hoffmann <m.hoffmann@cartelsol.com>
---
 test/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index 5c88c44..3aca80f 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -1,4 +1,4 @@
-PROJECT(mqtt-tests)
+PROJECT(mqtt-tests C)
 
 IF (WIN32)
   SET(MQTT_TEST_BROKER "tcp://mqtt.iotree.co.uk:1883" CACHE STRING "Hostname of a test MQTT broker to use")
-- 
2.1.4

